### PR TITLE
BEG-68 Bypass 2FA for Magento API for spesific Admin users

### PR DESCRIPTION
**Pull request by harshaaligent**
Add the "allowed_usernames" config to specify admin user names to bypass 2FA when the admin token generating with the old method

### CODE REVIEW

File: Helper/Data.php [hunk 1]
```diff
@@ -32,23 +51,54 @@ public function __construct(
...
         return $this->isBypassAllowed(self::ENV_VARIABLE_BYPASS_2FA_API);
     }
 
+    /**
+     * Validate the usernames that can be bypassed
+     *
+     * @param string $userName
+     * @return bool
+     */
+    public function isAllowedBypassAPiByUsername(string $userName): bool
```
Comment (aligent-lturner) [inline Helper/Data.php]: For consistency with other functions, can the `i` in API be made uppercase?

File: README.md [hunk 2]
```diff
@@ -32,7 +36,7 @@ By committing this setting to the codebase, it prevents someone from overriding
 ### Magento Cloud
 Magento Cloud environment-specific variables can be used to switch this feature on and off on a per cloud instance, especially 
 useful for integration environments where emails may not be configured.  First make sure that you've added 
-the hostname to `config.php` in the `allowed_hostnames` section, and then in the Cloud dashboard:
+the hostname to `config.php` in the `allowed_hostnames` section, and then also check you've added the usernames in the `allowed_usernames` section that need to be bypassed 2FA when generating the admin tokens. and then in the Cloud dashboard:
```
Comment (aligent-lturner) [inline README.md]: Grammar:
```
also check you've added... the admin tokens.
Then, in the Cloud dashboard:
```

**Review by aligent-lturner — COMMENTED** [review]
Just a couple of minor points

File: README.md (hunk 2 above)
Comment (harshaaligent) [reply README.md]: This is fixed

File: Helper/Data.php (hunk 1 above)
Comment (harshaaligent) [reply Helper/Data.php]: This is fixed

**Review by aligent-lturner — APPROVED** [review]

**Pull request merged**